frontend/src/components/InternProjects/InternButton.js

```diff
@@ -1,23 +1,26 @@
-import { Text, Flex } from "@chakra-ui/react";
+import { Text, Flex, } from "@chakra-ui/react";
 
 function InternBox() {
     return (
+        
```

Comment:
I think this is extraneous whitespace, but this is too trivial to flag. In practice, software development teams would configure linters with settings to enforce a consistent style. However, we're operating on a scale too small for that.